lite/examples/speech_commands/android/app/src/main/java/org/tensorflow/lite/examples/speech/RecognizeCommands.java

```diff
@@ -123,14 +123,10 @@ public RecognitionResult processLatestResults(float[] currentResults, long curre
       }
     }
 
-    // Add the latest results to the head of the queue.
-    previousResults.addLast(new Pair<Long, float[]>(currentTimeMS, currentResults));
```

Comment:
The only change you're making seems to be effectively adding `.clone()` to the `currentResults` array. Can you simply add it here, instead of introducing a new method?

Reply:
Hey, yep that's correct. the only functional change is adding `.clone()`. Extracting the logic to another method was done to allow unit test coverage to be added. Would you prefer this fix to be made without the unit test coverage?

Reply:
@yyoon I've raised https://github.com/tensorflow/examples/pull/233 with just the `.clone()` change and no UT if that's the route you'd prefer to take.